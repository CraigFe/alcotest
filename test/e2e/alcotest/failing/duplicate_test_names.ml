(** Ensure that suites with duplicate test names are rejected. *)

module Alcotest = Alcotest.V1

let () =
  Alcotest.run __FILE__
    [
      ("duped", [ Alcotest.test_case "1" `Quick (fun () -> assert false) ]);
      ("duped", [ Alcotest.test_case "2" `Quick (fun () -> assert false) ]);
    ]
