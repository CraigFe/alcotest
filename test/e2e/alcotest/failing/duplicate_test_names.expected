duplicate_test_names.exe: internal error, uncaught exception:
                          Alcotest_core.Core.Registration_error("Duplicate test name: duped")
                          
